zerver/webhooks/statuspage/doc.md

```diff
@@ -0,0 +1,22 @@
+Get notifications from Statuspage.io in Zulip whenever any
+company that you subscribed to creates or updates an incident, or updates a component's
+status.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+1. What company needs to do:
+Go to the **subscribers settings** in **notifications** tab and **enable** notifications delivery through **webhook**.
+
+![](/static/images/integrations/statuspage/001.png)
+
+2. What users need to do:
+Go to the company's statuspage.io site(which is example.statuspage.io) and click **SUBSCRIBE TO UPDATES** button and
+provide there the Webhook URL.
+
```

Comment:
Okay, so replace the above point with this:
```
In order to subscribe to a company's Statuspage.io notifications:

1. Go to the company's Statuspage.io site (for instance, `example.statuspage.io`).
2. Enter the webhook URL constructed above in the **Target Webhook** field.
2. Click on **SUBSCRIBE TO UPDATES**.
```

For future reference, with instructions such as these, try to make sure that they are in the correct order. For example, you can't click on **SUBSCRIBE TO UPDATES** without providing the webhook URL first, etc. :)